test/e2e/framework/pod/wait.go

```diff
@@ -218,6 +218,7 @@ func WaitForPodsRunningReady(c clientset.Interface, ns string, minPods, allowedN
 			default:
 				if metav1.GetControllerOf(&pod) == nil {
 					e2elog.Logf("Pod %s is Failed, but it's not controlled by a controller", pod.ObjectMeta.Name)
+					notReady++
```

Comment:
why do we take this pod into account if we ignore it?

Reply:
I intended to have it be printed on the below error message (`Number of not-ready pods ...`), but had not noticed this change impacts the result of the test.

Reply:
Considering these pods are ignored, would it be correct to suppress the `Number of not-ready pods...` message entirely when the only pods that were in a not ready state are the ignored ones? I believe this was the original situation described in the linked issue.